src/azsdkdemoprocessor/src/main/java/demo/sdk/az/App.java

```diff
@@ -0,0 +1,93 @@
+package demo.sdk.az;
+
+import ch.qos.logback.classic.Logger;
+import com.azure.identity.*;
+import com.azure.storage.queue.*;
```

Comment:
nit: enumerate all imports instead of using `*`.

Reply:
okay, that was VS Code default behavior